tests/test_cli.py

```diff
@@ -115,3 +114,27 @@ def test_azure_images_offline(mock_azure_image_versions, runner):
         assert list(image.keys()) == expected_keys
 
     assert result.exit_code == 0
+
+@pytest.mark.e2e
+def test_gcp_images_live(runner):
+    """Run a live test against the Google Cloud API to get images via CLI."""
+    result = runner.invoke(cli.gcp_images)
+    parsed = json.loads(result.output)
+
+    assert isinstance(parsed, list)
+
+    assert {image["status"] for image in parsed} != "DEPRICATED"
```

Comment:
Just a tiny nitpick, I think it's "deprecate" other than that it looks great :)

Reply:
Ah thanks, didn't see that :sweat_smile: